app/models/provider.rb

```diff
@@ -5,6 +5,7 @@ class Provider < ApplicationRecord
   include EmsRefresh::Manager
   include SupportsFeatureMixin
   include TenancyMixin
+  include UuidMixin
```

Comment:
Was this missing? :weary: Do we need a data migration for this, then?

Reply:
yep :disappointed:

Reply:
Wow 😮